test/common/http/http1/codec_impl_test.cc

```diff
@@ -3438,6 +3440,24 @@ TEST_P(Http1ClientConnectionImplTest, ManyResponseHeadersRejected) {
   EXPECT_EQ(status.message(), "http/1.1 protocol error: headers count exceeds limit");
 }
 
+// Tests that the number of response headers is configurable.
+TEST_P(Http1ClientConnectionImplTest, LargeResponseHeadersConfigurable) {
+  max_response_headers_kb_ = 8192;
```

Comment:
I think the test should have a smaller value, just in case the underlying codec rejects larger values.

Reply:
Changed it to **4096** instead.